lib/mix_test_interactive/settings.ex

```diff
@@ -109,6 +110,18 @@ defmodule MixTestInteractive.Settings do
     |> Map.put(:patterns, patterns)
   end
 
+  @doc """
+  Provide a list of tag patterns.
+
+  Only tests matching one or more of the tags will be run.
+  """
+  @spec only_tags(t(), [String.t()]) :: t()
+  def only_tags(settings, tags) do
+    settings
+    |> all_tests()
```

Comment:
I think this is OK for now, but it raises a question about how patterns and tags should interact.

When I use the `p` command, should a subsequent `t` command clear out my patterns, or maintain them? And vice versa, should a pattern clear out any tags?